src/shared/components/oncoprint/OncoprintUtils.ts

```diff
@@ -166,6 +181,86 @@ export function getHeatmapTrackRuleSetParams(molecularAlterationType: string) {
     };
 }
 
+function getTreatmentTrackRuleSetParams(trackSpec: IHeatmapTrackSpec):RuleSetParams {
```

Comment:
could you just add a test for this?

Reply:
Ok, added tons of unit tests that test all edge cases. Again, updated the function under test to make all tests work.